### PR TITLE
DB-1412: use up to 4 process for e10s by default in Cliqz browser

### PR DESCRIPTION
**Pull request by alver-cliqz**

### CODE REVIEW

**Comment by alver-cliqz** [comment]
@luciancor, is it what we want to turn on, right?

File: cliqz.cfg [hunk 1]
```diff
@@ -42,6 +42,9 @@ lockPref("distribution.version", AppConstants.MOZ_APP_VERSION_DISPLAY);
 
 pref("browser.uitour.enabled", false);
 
+// Use up to 4 processes for webpages rendering (e10s feature)
```
Comment (luciancor) [inline cliqz.cfg]: it is called e10s-multi - but yes, this should do the trick

Comment (alver-cliqz) [inline cliqz.cfg]: yes, it do, i checked on my PC

**Comment by chrmod** [comment]
we should start to move on from `cliqz.cfg`, to some more standard location.

@alver-cliqz how about `mozilla-release/browser/app/profile/firefox.js` ?

**Comment by alver-cliqz** [comment]
@chrmod i don't like idea to mix with original FF files, because of merge. But we can discuss this, what we can improve here.

**Pull request merged**